next/platform/macos/macos.cmake

```diff
@@ -165,24 +165,34 @@ add_executable(
     ${MBGL_ROOT}/platform/default/src/mbgl/test/main.cpp
 )
 
+target_include_directories(
+    mbgl-test-runner
+    PUBLIC ${MBGL_ROOT}/include ${MBGL_ROOT}/test/include
+)
+
 target_compile_definitions(
     mbgl-test-runner
     PRIVATE WORK_DIRECTORY=${MBGL_ROOT}
 )
 
 target_link_libraries(
     mbgl-test-runner
-    PRIVATE mbgl-compiler-options mbgl-test
+    PRIVATE mbgl-compiler-options -Wl,-force_load mbgl-test
```

Comment:
Could you please tell why linker options were changed?

Reply:
Benchmarks and (g)tests are now built as static libraries. They use a macro to instantiate the tests and if the static library is not linked with the force flag the linker will not include all test classes.